app.py

```diff
@@ -23,4 +38,25 @@ def q1(g, w):
     g = Graph()
     w = generate_weights(g.E)
     g.print_graph(w)
-    q1(g, w)
+    r = g.V[random.randint(0, len(g.V) - 1)]  # pick random start
+    q1(g, w, r)
+
+    src = random.randint(0, N - 1)
+    dest = random.randint(0, N - 1)
+    while (dest in g.adj[src] or src == dest):           
+        src = random.randint(0, N - 1)
+        dest = random.randint(0, N - 1)     
+    
+    # make new edge the most 'expensive' for both veritces
```

Comment:
This is the most expensive weight in the whole graph, not just these both vertices

Reply:
i changed the comment